controllers/drcluster_controller.go

```diff
@@ -235,24 +248,25 @@ func (r DRClusterReconciler) processFencing(u *drclusterUpdater) (ctrl.Result, e
 	return ctrl.Result{Requeue: requeue}, u.statusUpdate()
 }
 
-type drclusterUpdater struct {
-	ctx        context.Context
-	object     *ramen.DRCluster
-	client     client.Client
-	log        logr.Logger
-	reconciler *DRClusterReconciler
-	mwUtil     util.MWUtil
+type drclusterInstance struct {
```

Comment:
Ok, as in past comments, the updater was for status updates only as in DRPolicy as well. Is there a reason to change it?

Reply:
Tried to make it similar to VRG does where there is a single VRGInstance struct under which VRG has its methods.

This way seemed easier as it reduces the necessity to maintain 2 structures. Please let me know if you prefer to have it similar to DRPolicy.